x-pack/plugins/observability_onboarding/server/routes/custom_logs/get_logs_count.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient } from '@kbn/core/server';
+
+export async function getLogsCount({
+  dataset,
+  namespace,
+  esClient,
+}: {
+  dataset: string;
+  namespace: string;
+  esClient: ElasticsearchClient;
+}) {
+  try {
+    const { hits } = await esClient.search({
+      index: `logs-${dataset}-${namespace}`,
+      terminate_after: 1,
```

Comment:
The function is called `getLogsCount` which is slightly misleading because it will always return a number between 0 and up to 1 per shard. Should it instead be called `getHasLogs` and return a boolean?